lib/rucio/common/schema/cms.py

```diff
@@ -339,6 +339,8 @@
                      "type": "string",
                      "pattern": r'^[a-z0-9-_]{1,30}$'}
 
+SCOPE_NAME_REGEXP = '/([^/]*)(?=/)(.*)'
```

Comment:
Doesn't this add a `/` in front of every `name`, even if it is not wanted?

Reply:
All names in CMS start with a / (See name regular expression in schema).